### ISSUE TITLE
[Forms] There is an error when compiling SCSS in Angular application 

### ISSUE THREAD

**Issue by yanmariomenev** [telerik/kendo-themes#2728]
**Describe the bug**
The following error happens when compiling SCSS in Angular application:

PS C:\user\projectName> ng b
✔ Browser application bundle generation complete.
✔ Copying assets complete.
⠋ **Generating index html...1 rules skipped due to selector errors:
  .k-input-label:dir(rtl) -> unmatched pseudo-class :dir**
✔ Index html generation complete.

For reference - support ticket 1533089

**To reproduce**
Check the referenced support ticket.

**Affected package (please remove the unneeded items)**
* theme-default
* theme-bootstrap
* theme-material

**Affected suites (please remove the unneeded items)**
* Kendo UI for Angular

**Comment by joneff** [comment]
TLDR: `:dir()` is a valid selector and should not be causing an erorr.

**A longer version**

The selector is intended to be like `dir` attribute, but with computed (including inherited) value. To quote:

> The :dir CSS pseudo-class matches elements based on the directionality of the text contained in it. Pseudo-class :dir() is not equivalent of using the [dir=…] attribute selector. The latter matches a value of the dir and doesn't match when no attribute is set, even if in that case the element inherits the value of its parent. On the other hand, :dir() will match the value calculated by the UA, being inherited or the auto value.

The selector is fully supported in Firefox for a while now and the implementation in Chrome (well Blink rendering engine) is implemented, but behind a flag.

Now back to the issue: depending on exact toolchain, said warning may appear. For instance, if the toolchain analyzes the application tree and sees if the selector is matched anywhere, as long as you there is no RTL content on the page, it won't match.

Alternatively, if there is a tool based on css selectors against a browser and that browser is not Firefox, considering the selector is not supported in chrome, again, no matches will be found.

Or it could be any linting tool with missing references ...

Whatever the cause for the error, `:dir()` pseudo selector is valid and should not be causing errors by itself.

See:
* https://developer.mozilla.org/en-US/docs/Web/CSS/:dir
* https://caniuse.com/css-dir-pseudo
* https://bugs.chromium.org/p/chromium/issues/detail?id=576815
